src/kOS.Safe/Compilation/Opcode.cs

```diff
@@ -744,7 +744,7 @@ public class OpcodeSetMember : Opcode
         public override void Execute(ICpu cpu)
         {
             Structure value = cpu.PopStructureEncapsulated();         // new value to set it to
-            string suffixName = cpu.PopStack().ToString().ToUpper();  // name of suffix being set
+            string suffixName = cpu.PopStack().ToString();            // name of suffix being set
```

Comment:
Looking at the reference source (which may or may not be how Mono does it) It looks like we should be in the clear.  If the strings are both ASCII, OrdinalIgnoreCase calls an unmanaged method without doing a string conversion.  Which means even if it does allocate the additional string, it won't hit the GC.  If they aren't both ASCII, you can see the different implementations at these two links:

https://referencesource.microsoft.com/#mscorlib/system/string.cs,786821813d8a6340

https://referencesource.microsoft.com/#mscorlib/system/string.cs,8711fff131bc4d0e

I'll look at ILSpy a little later